src/components/crossDevice/ClientSessionLinked/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { h, FunctionComponent } from 'preact'
+import classNames from 'classnames'
+import { Button } from '@onfido/castor-react'
+
+import ScreenLayout from '../../Theme/ScreenLayout'
+import PageTitle from '../../PageTitle'
+import { trackComponent } from '../../../Tracker'
+import { useLocales } from '~locales'
+import theme from '../../Theme/style.scss'
+import style from './style.scss'
+
+import type { StepComponentBaseProps } from '~types/routers'
+
+type Props = {
+  nextStep: () => void
```

Comment:
Maybe I was wrong to add this redundant `nextStep` somewhere else (like [this](https://github.com/onfido/onfido-sdk-ui/blob/9b2d8eb225770e55822b05cd31c97c65ef2985aa/src/components/FaceVideo/Intro.tsx#L19-L21)), but I believe you can use `StepComponentBaseProps` directly because `nextStep` was already in that type: https://github.com/onfido/onfido-sdk-ui/blob/9b2d8eb225770e55822b05cd31c97c65ef2985aa/src/types/routers.ts#L95

Reply:
Ah, I see. I happened to see it done that way in a random TS component file I opened so just went with it. Will try with `StepComponentBaseProps` directly instead.

Reply:
Looks like it isn't possible. There is a [TS error](https://github.com/onfido/onfido-sdk-ui/pull/1534/checks?check_run_id=3370992615#step:5:184) that `Object is possible 'undefined'` when I do this. 🤔

Reply:
Humm, kinda weird. Anw we can try to get rid of this with another PR.